libraries/protobuf/serialization/main.py

```diff
@@ -7,7 +7,7 @@
         print("Retrieve Sensor object from sensor.data")
         sensor = Sensor()
         sensor.ParseFromString(content)
-        print(f"Sensor name: {sensor.name}")
-        print(f"Sensor temperature: {sensor.temperature}")
-        print(f"Sensor humidity: {sensor.humidity}")
+        print("Sensor name: {}".format(sensor.name))
```

Comment:
Now we are using py3, right? I'm not asking you to change this back to f-string unless you want to... this was your example ;D

Reply:
You are right :man_facepalming:

Reply:
done